components/file_attachment_list/file_attachment_list.test.tsx

```diff
@@ -114,4 +115,41 @@ describe('FileAttachmentList', () => {
         expect(wrapper.find(SingleImageView).exists()).toBe(false);
         expect(wrapper.find(FileAttachment).exists()).toBe(true);
     });
+
+    test('should render plugin previews if some are registered', () => {
+        const AudioCmp = ((props) => <p>{props.fileInfo.name}</p>) as typeof FileAttachment;
+        const VideoCmp = ((props) => <p>{props.fileInfo.extension}</p>) as typeof FileAttachment;
```

Comment:
This type cast seems strange. This component is not a `FileAttachment`, so why are we casting it this way?